### DISCUSSION TITLE
Vercel's Domain Redirection for "now.sh" Suddenly Stopped Working

### DISCUSSION THREAD

**Discussion by vincentdoerig** [vercel/community#7300] [Feature Question] [answered]
### Question

I'm running a moderately sized open-source project (2.7k+ stars) called [LaTeX.css](https://github.com/vincentdoerig/latex-css), which has been hosted on Vercel for four years. The project was created during the time when Vercel was known as Zeit, with our website being hosted on a "now.sh" subdomain (latex.now.sh).

Since its first release, I hadn't altered the domain name, relying on Vercel's automatic redirection to their new "vercel.app" domain after the rebranding. However, today I received a report from a user indicating that the "latex.now.sh" domain is no longer resolving. This disruption is significant, as the domain was widely used in the project's documentation. The lack of response from this domain implies that numerous projects loading the stylesheet by linking `https://latex.now.sh/style.css` will break. To my knowledge, Vercel has neither notified me/the community about this change nor provided information on their website.

I'm looking to understand whether this is a permanent discontinuation and whether there was any prior announcement. Further, I'm unable to access Vercel's transitional documentation from 2020 (https://www.notion.so/vercel/Vercel-Launch-1cab2369ab1c4d8db1987b68708be683 and https://www.notion.so/Change-Timeline-fab7622780694753a92c0a396cea7eb6), making it difficult to ascertain the redirection's status. I had assumed such redirection would be maintained indefinitely.

I would greatly appreciate any clarification or explanation regarding this matter. Thank you.

**Comment by pawlean** [comment]
Hi @vincentdoerig!

Thanks for reporting.

> I'm looking to understand whether this is a permanent discontinuation and whether there was any prior announcement. 

No, we have not permanently discontinued the `now.sh` subdomain. We're currently investing further, I'll keep you updated in this thread.

**Reply by pawlean** [reply, marked as answer]
**Update**: We found an issue with the subdomain which has now been resolved and the domain should start working soon. Keep me updated, @vincentdoerig! Thanks again for your patience 🙏🏼

**Reply by vincentdoerig** [reply]
Awesome, thank you for the quick response. I can confirm that the domain is reachable again and the redirection is working as it previously did.

**Reply by pawlean** [reply]
Great news! Let us know if you have any more questions. 😄

[a comment by an automated account is not shown]
